### DISCUSSION TITLE
Real-time chat

### DISCUSSION THREAD

**Discussion by Armaldio** [greenworksjs/greenworks#7] [Ideas]
It could be great to have a chat like discord where all users can discuss about greenworks
Like not all discussions fit into GH Discussions

That can be Discord, Gitter or anything

I can still recommand Discord because:
- I'm used to it
- If the community grow, the tool does the job

Edit: just created Discord, not definitive: **https://discord.gg/vKauzHHh**

**Comment by ceifa** [comment]
Discord please!
Greenheart discord already have a channel for greenworks, maybe we can use it.

**Reply by Armaldio** [reply]
We are on our own now, no Greenheart

**Reply by sleepingpandagames** [reply]
Can you share the invite link? I would love to join and follow discussions along.

**Reply by Armaldio** [reply]
https://discord.gg/vKauzHHh
